vochain/transaction.go

```diff
@@ -185,54 +185,139 @@ func (app *BaseApplication) AddTx(vtx *VochainTx, commit bool) (*AddTxResponse,
 			return response, app.State.AddToRollingCensus(tx.ProcessId, tx.NewKey, weight)
 		}
 
-	case *models.Tx_SetAccountInfo:
-		txValues, err := SetAccountInfoTxCheck(vtx.Tx, vtx.SignedBody, vtx.Signature, app.State)
-		if err != nil {
-			return nil, fmt.Errorf("setAccountInfoTxCheck: %w", err)
+	case *models.Tx_SetAccount:
+		tx := vtx.Tx.GetSetAccount()
+		var err error
```

Comment:
I think we don't need to declare this error var here

Reply:
its a minor i fixed on #622 but lets fix it here too :)